Code/GraphMol/Canon.cpp

```diff
@@ -23,7 +23,7 @@ bool isUnsaturated(const Atom *atom, const ROMol &mol) {
        boost::make_iterator_range(mol.getAtomBonds(atom))) {
     // can't just check for single bonds, because dative bonds also have an
     // order of 1
-    if (mol[bndItr]->getBondTypeAsDouble() > 1.001) {
+    if (mol[bndItr]->getBondTypeAsDouble() > 1) {
```

Comment:
Still should be a double value if only to silence warning :)

Reply:
That actually doesn't produce a warning on any the platforms where we build.
There's really no reason to warn on `double > int` or `double > long int`